src/proof/alethe/alethe_proof_rule.h

```diff
@@ -0,0 +1,416 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Hanna Lachnitt
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Enumeration of Alethe proof rules
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC4__PROOF__ALETHE_PROOF_RULE_H
+#define CVC4__PROOF__ALETHE_PROOF_RULE_H
+
+#include <memory>
+
+namespace cvc5 {
+
+namespace proof {
+
+enum class AletheRule : uint32_t
+{
+  //================================================= Special Rules: Commands
+  // These rules should be printed as commands
+  //
+  // ======== subproof
+  // > i1. F1
```

Comment:
Since you are using some particular notation, you should describe it briefly. Say that you're writing the doc as a series of steps with IDs, a context and that a context may have substitutions ("->"). And that more information can be found on the $link_to_Alethe_spec.